projects/aca-playwright-shared/src/api/file-actions.ts

```diff
@@ -50,7 +50,13 @@ export class FileActionsApi {
     });
   }
 
-  async uploadFileWithRename(fileLocation: string, newName: string, parentId: string = '-my-', title: string = '', description: string = '') {
+  async uploadFileWithRename(
+    fileLocation: string,
+    newName: string,
+    parentId: string = '-my-',
+    title: string = '',
+    description: string = ''
+  ): Promise<NodeEntry | null> {
```

Comment:
If you want to return Promise I wouldn't just return null, you can return rejected promise instead and catch it whenever you use the method

Reply:
removed and added reject promise